app/src/main/res/layout/layout_pending_safe_item.xml

```diff
@@ -1,23 +1,51 @@
 <?xml version="1.0" encoding="utf-8"?>
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:background="@drawable/selectable_background"
+    android:background="@color/text_grey_background"
     android:gravity="center_vertical"
     android:padding="16dp">
 
     <me.zhanghai.android.materialprogressbar.MaterialProgressBar
-        android:layout_width="48dp"
-        android:layout_height="48dp"
-        android:padding="8dp" />
+        android:layout_width="40dp"
+        android:layout_height="40dp"
+        android:background="@drawable/progress_circle_background"
+        android:padding="8dp"
+        android:theme="@style/WhiteCircleProgress"
+        app:backgroundTint="@color/gnosis_blue" />
 
-    <TextView
-        android:id="@+id/layout_pending_safe_item_name"
-        android:layout_width="wrap_content"
+    <LinearLayout
+        android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:ellipsize="end"
-        android:maxLines="1"
-        style="@style/Title"
-        tools:text="My Safe" />
+        android:layout_marginLeft="16dp"
+        android:layout_marginStart="16dp"
+        android:orientation="vertical">
+
+        <TextView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="@string/deploying_new_safe"
+            android:textColor="@color/gnosis_blue"
+            android:textSize="15sp"
+            android:textStyle="bold" />
+
+        <TextView
+            android:id="@+id/layout_pending_safe_item_name"
+            style="@style/Title"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:ellipsize="end"
+            android:maxLines="1"
+            android:textColor="@color/gnosis_dark_blue"
+            android:textSize="15sp"
```

Comment:
Same here (just Content)

Reply:
I would remove the textSize too, as it should be applied by the style (same for all)